src/api/app/views/webui/user/_index_actions.html.haml

```diff
@@ -1,6 +1,4 @@
 - content_for :actions do
-  - if configuration.accounts_editable?(user) && !feature_enabled?('user_profile_redesign') && policy(user).update?
```

Comment:
Users can edit their account with the `Edit` link just above their avatar, this code can be safely deleted.